activiti-cloud-services-audit-jpa/src/main/java/org/activiti/cloud/services/audit/jpa/converters/ActivityCancelledEventConverter.java

```diff
@@ -2,42 +2,40 @@
 
 import org.activiti.api.process.model.events.BPMNActivityEvent;
 import org.activiti.cloud.api.model.shared.events.CloudRuntimeEvent;
+import org.activiti.cloud.api.model.shared.impl.events.CloudRuntimeEventImpl;
 import org.activiti.cloud.api.process.model.events.CloudBPMNActivityCancelledEvent;
 import org.activiti.cloud.api.process.model.impl.events.CloudBPMNActivityCancelledEventImpl;
-import org.activiti.cloud.services.audit.api.converters.EventToEntityConverter;
 import org.activiti.cloud.services.audit.jpa.events.ActivityCancelledAuditEventEntity;
 import org.activiti.cloud.services.audit.jpa.events.AuditEventEntity;
-import org.springframework.stereotype.Component;
 
-@Component
-public class   ActivityCancelledEventConverter implements EventToEntityConverter<AuditEventEntity> {
+public class ActivityCancelledEventConverter extends BaseEventToEntityConverter {
```

Comment:
@CTI777 I created a base event converter that will set the extra info. All the converters should extend this `BaseEventToEntityConverter` now.